### PR TITLE
qa/suites/rados/cephadm/thrash: add cephadm thrashing test

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: Sage Weil <sage@newdream.net>

### CODE REVIEW

File: qa/suites/rados/cephadm/thrash/0-distro/ubuntu_18.04_podman.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+.qa/distros/all/ubuntu_18.04_podman.yaml
```
Comment (sebastian-philipp) [inline qa/suites/rados/cephadm/thrash/0-distro/ubuntu_18.04_podman.yaml:1]: sure about adding ubuntu+podman?

Comment (liewegas) [reply qa/suites/rados/cephadm/thrash/0-distro/ubuntu_18.04_podman.yaml:1]: oops! fixing

**Comment by liewegas** [comment]
@neha-ojha @jdurgin do we want to add thrashing here, or should we be transitioning the normal thrashing tests to use cephadm?

**Comment by neha-ojha** [comment]
> @neha-ojha @jdurgin do we want to add thrashing here, or should we be transitioning the normal thrashing tests to use cephadm?

I think we'll want the normal thrashing tests to use cephadm if we can get them to pass with some level of confidence. Maybe worth adding a small subset of thrashing tests within rados/cephadm and stabilize them before we transition.

**Comment by sebastian-philipp** [comment]
> adding a small subset of thrashing tests within rados/cephadm and stabilize them before we transition.

That would be great.

**Comment by liewegas** [comment]
retest this please

**Comment by liewegas** [comment]
retest this please

**Comment by liewegas** [comment]
retest this please

**Review by neha-ojha — APPROVED** [review]

**Review by sebastian-philipp — APPROVED** [review]

**Pull request merged**